### PR TITLE
Change from jar to directory

### PR DESCRIPTION
**Pull request by arix00**
change from chrome.jar to directory for easy developing

### CODE REVIEW

File: src/make.sh [hunk 1]
```diff
@@ -9,17 +9,12 @@ ZIPARGS=-r
...
 
 echo "Cleaning"
-rm -f chrome/$CHROMEPACKAGE
 rm -f $PACKAGE
 
 echo "Packaging using version $VERSION"
-echo "Package chrome jar"
-cd chrome && $ZIP $ZIPARGS $CHROMEPACKAGE $CHROMEFILES > ../../chromepackage.out
 echo "Packaging XPI $PACKAGE"
 cd .. && $ZIP $ZIPARGS $PACKAGE $FILES > ../xpipackage.out
```
Comment (ath0mas) [inline src/make.sh]: `cd .. &&` should be removed, as you do not `cd` into `chrome` anymore

**Comment by ath0mas** [comment]
@arix98 After **make.sh** fix, please change **makefile** the same way to have both build scripts up to date

**Comment by arix00** [comment]
@ath0mas OK make file updated. Please review.

**Comment by ath0mas** [comment]
All right!
Please squash your commits into a single one (`git rebase -i`) ; then I'll merge ;)

**Comment by arix00** [comment]
@ath0mas ok done.

**Pull request merged**